### PR TITLE
add 58 length-of-last-word

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m

### CODE REVIEW

File: 58.length-of-last-word.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+
+# @lc code=start
+class Solution:
+    def lengthOfLastWord(self, s: str) -> int:
+        """
+        time: O(1)
+        space: O(len(l))  l : number of words
+        """
+        l = s.split()
```
Comment (ryuji0123) [inline 58.length-of-last-word.py:14]: 'l' is unclear

File: 58.length-of-last-word.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        if not l:
```
Comment (ryuji0123) [inline 58.length-of-last-word.py:15]: return len(l[-1]) if l else 0

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I fixed my code.  Please check it.

**Review by ryuji0123 — APPROVED** [review]
LGTM

**Pull request merged**